### DISCUSSION TITLE
buscando equipo

### DISCUSSION THREAD

**Discussion by AlvaroFierro** [Launch-X-Latam/MisionFrontEnd#548] [Team Posts]
# Busco equipo para el hackathon

alguien que no tenga equipo, o que quiera armar uno?

**Comment by jlnunezobando** [comment]
No tengo grupo, si crean uno cuenten conmigo lx08651@innovaccion.mx

**Comment by jeymarherrera** [comment]
yo, mi correo es este lx06608@innovaccion.mx

**Comment by AlexG0712** [comment]
Me faltan 2 para completar el equipo, lx04297@innovaccion.mx

**Comment by JoseEduardoVelazquezCN** [comment]
***@***.*** yo aún no tengo equipo puedo entrar en el suyo ¿?

Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by AndresLunaH** [comment]
Hola, si aún tienes espacio para uno me encantaría unirme, mi correo es: lx15493@innovaccion.mx

**Comment by ALISONQS** [comment]
> hackathon

Hola Andres yo tambien busco grupo ¿te gustaría hacer equipo?

**Comment by Evelo00** [comment]
Hola, si alguien quiere agregarme a su equipo esta perdecto :) tengo conocimientos en Html - CSS  y un poco de js.
lx07667@innovaccion.mx <--- pueden contactarme ahí
